xml/System.Runtime.CompilerServices/Unsafe.xml

```diff
@@ -486,10 +598,10 @@
         <Parameter Name="source" Type="System.Void*" />
       </Parameters>
       <Docs>
-        <typeparam name="T">The type of the interpreted location.</typeparam>
-        <param name="source">The location of the value to reference.</param>
-        <summary>Reinterprets the given location as a reference to a value of type <typeparamref name="T" />.</summary>
-        <returns>A reference to a value of type <typeparamref name="T" />.</returns>
+        <typeparam name="T">The elemental type of the managed pointer.</typeparam>
+        <param name="source">The unmanaged pointer to convert.</param>
+        <summary>Converts an unmanaged pointer into a managed pointer to a value of type <typeparamref name="T" />.</summary>
+        <returns>A managed pointer to a value of type <typeparamref name="T" />.</returns>
         <remarks>To be added.</remarks>
```

Comment:
This deserves a remark that it is the callers responsibility to ensure the pointer is properly aligned for `T`. The JIT assumes byrefs are aligned by default.